components/feature/pwa/src/main/java/mozilla/components/feature/pwa/ManifestStorage.kt

```diff
@@ -53,10 +71,27 @@ class ManifestStorage(context: Context) {
         }
     }
 
+    /**
+     * Update the last time a web app was used.
+     *
+     * @param url an url that is part of an installed web app
+     */
+    suspend fun updateManifestUsedAt(url: String) = withContext(IO) {
```

Comment:
I think this can just the `startUrl` instead of the scope, since you'll only update the manifest in a running PWA. It should just take the manifest as a parameter to avoid confusion when calling the function.